axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseLineService.java

```diff
@@ -18,8 +18,17 @@
  */
 package com.axelor.apps.hr.service.expense;
 
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.base.db.Product;
+import com.axelor.apps.hr.db.Employee;
 import com.axelor.apps.hr.db.Expense;
 import com.axelor.apps.hr.db.ExpenseLine;
+import com.axelor.apps.hr.db.KilometricAllowParam;
+import com.axelor.apps.project.db.Project;
+import com.axelor.meta.db.MetaFile;
+import java.math.BigDecimal;
+import java.time.LocalDate;
 import java.util.List;
 
 public interface ExpenseLineService {
```

Comment:
Please create instead new services, one dedicated to the creation of expense line and another for convertProofFilesInPdf